homeassistant/components/thermostat/ecobee.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/local/bin/python3
```

Comment:
Can you remove this line, it's not needed because this file will not be executed directly.